assembl/static2/js/app/components/debate/brightMirror/instructionView.jsx

```diff
@@ -0,0 +1,61 @@
+// @flow
+import React from 'react';
+
+import Permissions, { connectedUserCan } from '../../../utils/permissions';
+import TopPostFormContainer from '../common/topPostFormContainer';
+import { getIfPhaseCompletedByIdentifier } from '../../../utils/timeline';
+import FictionsList from './fictionsList';
+import InstructionsText from './instructionsText';
+
+type Post = {
```

Comment:
@cedrtang I think we should use BrightMirrorFictionFragment when rebasing with DEVAS 1726